python/cuml/test/test_dbscan.py

```diff
@@ -50,18 +49,15 @@ def stress_param(*args, **kwargs):
 @pytest.mark.parametrize('ncols', [unit_param(3), quality_param(100),
                          stress_param(1000)])
 def test_dbscan(datatype, input_type, use_handle,
-                max_bytes_per_batch, nrows, ncols):
-    # max_bytes_per_batch sizes: 10=6 batches, 200=2 batches, 2e6=1 batch
+                nrows, ncols):
     n_samples = nrows
     n_feats = ncols
     X, y = make_blobs(n_samples=n_samples,
                       n_features=n_feats, random_state=0)
-    if nrows != 500000:
-        skdbscan = skDBSCAN(eps=3, min_samples=2)
-        sk_labels = skdbscan.fit_predict(X)
+
     handle, stream = get_handle(use_handle)
-    cudbscan = cuDBSCAN(handle=handle, eps=3, min_samples=2,
-                        max_bytes_per_batch=max_bytes_per_batch)
```

Comment:
I'm going to dive into this a bit further. I'm hesitant to remove this altogether because I believe this may be exposing a bug in dbscan that needs to be addressed.

Reply:
I can add it in with different max_bytes_per_batch values so we can run the tests without them getting killed. The dbscan code for stress test might just take a lot longer to run

Reply:
added it back in